MVPproject/MVPproject/Builders/ModuleBuilder.swift

```diff
@@ -57,11 +57,14 @@ final class ModuleBuilder {
     static func makeCategoryModule(coordinator: RecipesCoordinator, category: RecipesCategory) -> UIViewController {
         let viewController = CategoryViewController()
         let networkService = NetworkService()
+        let loadImageService = LoadImageService()
+        let loadImageProxy = LoadImageProxy(service: loadImageService)
         let presenter = CategoryPresenter(
             view: viewController,
             coordinator: coordinator,
             networkService: networkService,
-            category: category
+            category: category,
+            loadImgaeProxy: loadImageProxy
```

Comment:
вообще я бы назвала этот параметр как loadImageService
весь прикол, что презентер не в курсе, что он с прокси работает, он думает, что с сервисом. Если я конечно не путаю чего)
```suggestion
            loadImageProxy: loadImageProxy
```

Reply:
Да согл